Source/astcenc_weight_align.cpp

```diff
@@ -62,11 +73,15 @@ static const float angular_steppings[44] = {
 	20.0f, 21.0f, 22.0f, 23.0f,
 	24.0f, 25.0f, 26.0f, 27.0f,
 	28.0f, 29.0f, 30.0f, 31.0f,
-	32.0f, 33.0f, 34.0f, 35.0f
+	32.0f, 33.0f, 34.0f, 35.0f,
+#if ANGULAR_STEPS >= 48
+    // This is "redundant" and only used in more-than-4-wide
```

Comment:
Can we extend the comment with something along the lines of "The same value is replicated so that SSE and AVX code paths will return the same results.".